akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FormFieldDirectivesSpec.scala

```diff
@@ -168,58 +168,58 @@ class FormFieldDirectivesSpec extends RoutingSpec {
   "The 'formField' requirement directive" should {
     "block requests that do not contain the required formField" in {
       Post("/", urlEncodedForm) ~> {
-        formField('name ! "Mr. Mike") { completeOk }
+        formField("name".requiredValue("Mr. Mike")) { completeOk }
```

Comment:
I found `requiredValue` a bit confusing. How about something like:

```scala
formField("name".shouldBe("Mr. Mike"))
formField("name".whenValued("Mr. Mike"))
formField("name".ifValued("Mr. Mike"))
formField("name".valued("Mr. Mike"))
formField("name".is("Mr. Mike")) // I think this is my preferred
```
so it looks like:
```scala
Post("/", urlEncodedForm) ~> { 
   formField("name".is("Mr. Mike")){ 
      ...
   }
}
```